Tests/Kernels/GraphicTest/Kernel.cs

```diff
@@ -166,13 +166,13 @@ protected override void Run()
                 TestBitmaps();
 
                 /* First test with the DefaultMode */
-                Canvas canvas = FullScreenCanvas.GetFullScreenCanvas();
```

Comment:
Can we add some test with double buffered canvases?

Reply:
I will make a new pull request with simple buffered VBE canva and add some tests to it.